Makefile

```diff
@@ -3,21 +3,23 @@
 THIS_FILE := $(lastword $(MAKEFILE_LIST))
 CURRENT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
-#################################
+############################
 # Make targets for dsp-tools
-#################################
-
-.PHONY: clone-dsp-repo
-clone-dsp-repo: ## clone the dsp-api git repository
-	@git clone --branch main --single-branch --depth 1 https://github.com/dasch-swiss/dsp-api.git $(CURRENT_DIR)/.tmp/dsp-stack
+############################
 
 .PHONY: dsp-stack
-dsp-stack: ## run the dsp-stack (deletes existing volumes first)
-	$(MAKE) -C $(CURRENT_DIR)/.tmp/dsp-stack env-file
-	$(MAKE) -C $(CURRENT_DIR)/.tmp/dsp-stack stack-down-delete-volumes
-	$(MAKE) -C $(CURRENT_DIR)/.tmp/dsp-stack init-db-test
-	$(MAKE) -C $(CURRENT_DIR)/.tmp/dsp-stack stack-up
-	$(MAKE) -C $(CURRENT_DIR)/.tmp/dsp-stack stack-logs-api-no-follow
+dsp-stack: ## clone the dsp-api git repository and run the dsp-stack
+	@mkdir -p .tmp
+	@git clone --branch main --single-branch --depth 1 https://github.com/dasch-swiss/dsp-api.git .tmp/dsp-stack
+	$(MAKE) -C .tmp/dsp-stack env-file
+	$(MAKE) -C .tmp/dsp-stack init-db-test
+	$(MAKE) -C .tmp/dsp-stack stack-up
+	$(MAKE) -C .tmp/dsp-stack stack-logs-api-no-follow
+
+.PHONY: stack-down
+stack-down: ## stop dsp-stack and remove the cloned dsp-api repository
+	$(MAKE) -C .tmp/dsp-stack stack-down-delete-volumes
+	@sudo rm -rf .tmp
```

Comment:
Is it necessary to remove the tmp folder? Why not just leave it?

Reply:
`make dsp-stack` is the only place in the entire repo where the tmp folder is created. So it makes sense to me to delete it in the `stack-down` command, instead of the `make clean` command.